packages/react-widgets/src/models/utils.js

```diff
@@ -14,6 +14,9 @@ export function isRemoteCalculationSupported(props) {
 export function wrapModelCall(props, fromLocal, fromRemote) {
   const { source, global, remoteCalculation } = props;
 
+  // TODO CHECK THIS
+  if (source.geoColumn) return fromLocal(props);
```

Comment:
@Josmorsot do we need to add a different behaviour in the backport? I don't think so...